tests/test_forms.py

```diff
@@ -0,0 +1,58 @@
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm, DriverLicenseUpdateForm
+
+
+class DriverCreationFormTest(TestCase):
+    def setUp(self) -> None:
+        self.form_data = {
+            "username": "some_username",
+            "password1": "some_password123",
+            "password2": "some_password123",
+            "license_number": "ABC12345"
+        }
+
+    def test_form_is_valid(self) -> None:
+        form = DriverCreationForm(data=self.form_data)
+        print(form.errors)
```

Comment:
The print statement here is unnecessary for the test and should be removed to keep the test output clean.